src/main/java/it/gov/pagopa/payhub/activities/connector/organization/client/BrokerClient.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.payhub.activities.connector.organization.client;
+
+import it.gov.pagopa.payhub.activities.connector.organization.config.OrganizationApisHolder;
+import it.gov.pagopa.pu.organization.dto.generated.PagedModelBroker;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Lazy
+@Service
+public class BrokerClient {
+	private final OrganizationApisHolder organizationApisHolder;
+
+	public BrokerClient(OrganizationApisHolder organizationApisHolder) {
+		this.organizationApisHolder = organizationApisHolder;
+	}
+
+	public PagedModelBroker fetchAll(String accessToken) {
+		return organizationApisHolder.getBrokerEntityControllerApi(accessToken).crudGetBrokers(null, null, null);
```

Comment:
null doesn't mean no pagination, but it will ask for default values!
(if I remember, page 0 and size 20)

you have to explicitly set this values